src/cavendish_particle_tracks/_decay_angles_dialog.py

```diff
@@ -17,27 +20,22 @@
 class DecayAnglesDialog(QDialog):
     def __init__(self, parent=None):
         super().__init__(parent)
-
         self.parent = parent
-
+        # region UI Setup
         self.setWindowTitle("Decay Angles")
-
         # text boxes for track parameters and results
         self.textboxes_slope = [QLabel(self) for _ in range(3)]
         self.textboxes_intercept = [QLabel(self) for _ in range(3)]
         self.textboxes_phi = [QLabel(self) for _ in range(2)]
-
         for textbox in (
             self.textboxes_slope + self.textboxes_intercept + self.textboxes_phi
         ):
             textbox.setMinimumWidth(200)
-
-        bss = QPushButton("Calculate")
-        bss.clicked.connect(self._on_click_calculate)
-
-        bap = QPushButton("Save to table")
-        bap.clicked.connect(self._on_click_save_to_table)
-
+        # buttons
+        btn_calculate = QPushButton("Calculate")
+        btn_calculate.clicked.connect(self._on_click_calculate)
+        btn_save = QPushButton("Save to table")
+        btn_save.clicked.connect(self._on_click_save_to_table)
```

Comment:
Changes are off-topic from the PR title.

Reply:
I'd keep them as some modifications to the UI _are_ needed.